src/gui/settings.cpp

```diff
@@ -719,105 +721,123 @@ void FurnaceGUI::drawSettings() {
       CONFIG_SECTION("Audio") {
         // SUBSECTION OUTPUT
         CONFIG_SUBSECTION("Output");
+        if (ImGui::BeginTable("##Output",2)) {
+          ImGui::TableSetupColumn("##Label",ImGuiTableColumnFlags_WidthFixed);
+          ImGui::TableSetupColumn("##Combo",ImGuiTableColumnFlags_WidthStretch);
 #ifdef HAVE_JACK
-        ImGui::AlignTextToFramePadding();
-        ImGui::Text("Backend");
-        ImGui::SameLine();
-        int prevAudioEngine=settings.audioEngine;
-        if (ImGui::Combo("##Backend",&settings.audioEngine,audioBackends,2)) {
-          if (settings.audioEngine!=prevAudioEngine) {
-            if (!isProAudio[settings.audioEngine]) settings.audioChans=2;
+          ImGui::TableNextRow();
+          ImGui::TableNextColumn();
+          ImGui::AlignTextToFramePadding();
+          ImGui::Text("Backend");
+          ImGui::TableNextColumn();
+          int prevAudioEngine=settings.audioEngine;
+          if (ImGui::Combo("##Backend",&settings.audioEngine,audioBackends,2)) {
+            if (settings.audioEngine!=prevAudioEngine) {
+              if (!isProAudio[settings.audioEngine]) settings.audioChans=2;
+            }
           }
-        }
 #endif
 
-        if (settings.audioEngine==DIV_AUDIO_SDL) {
+          ImGui::TableNextRow();
+          ImGui::TableNextColumn();
```

Comment:
Shouldn't this be inside `if (settings.audioEngine==DIV_AUDIO_SDL) {`?

Reply:
yes